tests/Aspire.Hosting.Tests/AddParameterTests.cs

```diff
@@ -43,54 +42,7 @@ public void ParametersAreHiddenByDefault()
             {
                 Assert.Equal(CustomResourceKnownProperties.Source, prop.Name);
                 Assert.Equal("Parameters:pass", prop.Value);
-            },
-            prop =>
-            {
-                Assert.Equal("Value", prop.Name);
-                Assert.Equal("pass1", prop.Value);
-            });
-    }
-
-    [Fact]
-    public void MissingParametersAreConfigurationMissing()
```

Comment:
Isn't this test still valid except for the hook bit at the bottom, or is it mostly redundant?

Reply:
The first test in this file tests the same things, this adds the hook.